### PR TITLE
Users update

### LINKED ISSUE #141: Restructure users with button, data and api changes
Refactor users:
 - [ ] update Users model with ERD
 - [ ] update api endpoints with activation and deactivation, but no deletion
 - [ ] update tests for users api
 - [ ] update epics page with button label and data imports

### PR DESCRIPTION
**Pull request by jkrobicki**
- added new attributes in users model
- corrected all imports to and from pages/users,py and data/users/py (it's the cause of so many files changed)
- added new utility `switch_state` in pages/utils.py
- updated sample.sql
closes #141

### CODE REVIEW

File: frontend/data/users.py [hunk 1]
```diff
@@ -1,15 +1,122 @@
 import requests
+import json
+from typing import TypedDict, List
+from datetime import datetime
 from config import base_url
+from components.input import Selector2, SelectorDropdownKeyValue
```
Comment (liberty-rising) [inline frontend/data/users.py:6]: Components are being imported into the data file. Should be moved to the page file.

File: frontend/data/users.py [hunk 2]
```diff
@@ -1,15 +1,122 @@
...
+        is_active=True,
+    )
+    response = requests.post(
+        f"{base_url}/api/users",
+        data=json.dumps(dict(data)),
+        headers={"accept": "application/json", "Content-Type": "application/json"},
+    )
+
+
+def user_dropdown(set_user_id):
```
Comment (liberty-rising) [inline frontend/data/users.py:59]: Supposed to be in the page file.

**Review by liberty-rising — COMMENTED** [review]
There is a component that still exists in the users data file when it should be in the user page file.

File: frontend/data/users.py (hunk 2 above)
Comment (jkrobicki) [reply frontend/data/users.py]: Corrected the issue, I removed the whole component as its duplicated. New commit with update was sent.

Comment (acivitillo) [reply frontend/data/users.py]: @liberty-rising could you review? If you approve, I will approve too and merge

Comment (liberty-rising) [reply frontend/data/users.py]: @acivitillo Looks good to go

**Review by acivitillo — APPROVED** [review]

**Pull request merged**